tools/bazel.rc

```diff
@@ -2,10 +2,36 @@
 
 build --workspace_status_command=bazel/get_workspace_status
 
-build:asan --copt -fsanitize=address
-build:asan --linkopt -fsanitize=address
+# Basic ASAN/UBSAN that works for gcc-4.9
+build:asan --copt -fsanitize=address,undefined
+build:asan --linkopt -fsanitize=address,undefined
 build:asan --linkopt -ldl
 build:asan --define tcmalloc=disabled
 build:asan --build_tag_filters=-no_asan
 build:asan --test_tag_filters=-no_asan
 build:asan --define signal_trace=disabled
+
+# Clang 5.0 ASAN
```

Comment:
Too many  unaligned pointer issues. We can fix these later if we care (certainly a performance advantage and more interesting as we consider running on more than just x86), I'll add a TODO.